### PR TITLE
Improve modal object without using this

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Code quality of modal wasnt at his best
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Noone
| How to test?  | Go on /admin-dev/index.php/sell/attachments/?_token=pMf66fXNkQOtMt9aWrFAuicO80ucFRUUhONYp9KpzvU , add some files and try to delete with bulk action, this pr is improvement of code quality of the modal

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17079)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

File: admin-dev/themes/new-theme/js/components/modal.js [hunk 1]
```diff
@@ -64,91 +63,96 @@ export default function ConfirmModal(params, confirmCallback) {
...
-function Modal({id = 'confirm_modal', confirmTitle, confirmMessage = '', closeButtonLabel = 'Close', confirmButtonLabel = 'Accept', confirmButtonClass = 'btn-primary'}) {
+function Modal(
+  {
+    id = 'confirm_modal',
+    confirmTitle,
+    confirmMessage = '',
+    closeButtonLabel = 'Close',
+    confirmButtonLabel = 'Accept',
+    confirmButtonClass = 'btn-primary',
+  }) {
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/components/modal.js:83]:
```suggestion
  }
) {
```

**Review by PierreRambaud — APPROVED** [review]

File: admin-dev/themes/new-theme/js/components/modal.js (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/js/components/modal.js:83]: ESLint refuse ça

Comment (PierreRambaud) [reply admin-dev/themes/new-theme/js/components/modal.js:83]: No prob :D

**Comment by Progi1984** [comment]
Thanks @NeOMakinG

**Pull request merged**